onnxruntime/test/onnx/main.cc

```diff
@@ -1153,6 +1259,43 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
     broken_tests.insert({"sce_sum_expanded", "result differs"});
     broken_tests.insert({"sce_sum_log_prob", "result differs"});
     broken_tests.insert({"sce_sum_log_prob_expanded", "result differs"});
+    // float 8, not implemented yet
+    broken_tests.insert({"cast_FLOAT8E5M2FNUZ_to_FLOAT", "float 8 not implemented yet"});
```

Comment:
as is, these are only added to `broken_tests` if `enable_qnn` is true. is that intended?

Reply:
Yes and no. I tried to disable the tests in many places to see the impact. My goal was to make CI pass and then remove one different test in every place I modified to understand which pipeline is impacted by which list and clean my changes.